public/react/TopToolbarTitle.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { CSSProperties } from "react";
+import { dashboardSidebarWidth, fontTitle, palette } from "./style";
+import { css } from "@emotion/react";
+
+interface Props {
+  title: string;
+}
+
+const titlePartCommon: CSSProperties = {
+  display: "inline-block",
+  textAlign: "center",
+  lineHeight: "55px",
+};
+
+const titleStyle = css({
+  height: '100%',
+  width: dashboardSidebarWidth,
+  minWidth: dashboardSidebarWidth,
+  fontSize: 18,
```

Comment:
My personal preference is to always give font sizes an explicit suffix - in this case `px`? - just in case

Reply:
👍 Makes sense - I managed to trip myself up earlier when I forgot that `px` is not the default unit for `lineHeight`, so all in favour of explicit units for typography.